visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -192,7 +191,7 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		const spriteMaterial = new SpriteMaterial({ map: texture })
 		const sprite = new Sprite(spriteMaterial)
 		this.lineCount = multiLineContext.length
-		this.setLabelSize(sprite, canvas.width, null)
+		this.setLabelSize(sprite, null, canvas.width)
```

Comment:
The order of label and width hast changed. If we haven't got a label we must pass `null`. We could specify the parameter as an optional one so that we can omit it.